install_config/router/f5_router.adoc

```diff
@@ -65,12 +65,17 @@ endif::[]
 * 11.x
 * 12.x
 
-However, the following features are not supported with *F5 BIG-IP®*:
+[IMPORTANT]
+====
+The following features are not supported with *F5 BIG-IP®*:
 
+* Wildcard routes together with re-encrypt - you must supply a certificate and key in the route. And if you provide a certificate, a key and a CA, the CA is never used.
```

Comment:
Should this be _Wildcard routes together with re-encryption_? Seems like something is missing here. The [3.6 release notes](https://github.com/openshift/openshift-docs/blob/f6a965f6af48b852f838f12160ca42aff3ffb5c7/release_notes/ocp_3_6_release_notes.adoc) talk about _re-encrypt routes_.

Reply:
s/a certificate, a key and a CA/a certificate, a key, and a CA ??